src/Struct/FindologicProduct.php

```diff
@@ -415,7 +415,7 @@ protected function getAttributeProperties(
             $group = $propertyGroupOptionEntity->getGroup();
             if ($group && $propertyGroupOptionEntity->getTranslation('name') && $group->getTranslation('name')) {
                 $properyGroupAttrib = new Attribute(
-                    Utils::removeControlCharacters($group->getTranslation('name')),
+                    Utils::cleanFilterName($group->getTranslation('name')),
```

Comment:
I would just go for `removeSpecialChars`, it takes a string a`preg_replace` and stuff.